net/instaweb/rewriter/public/rewrite_options.h

```diff
@@ -1527,6 +1528,13 @@ class RewriteOptions {
     set_option(x, &blink_blacklist_end_timestamp_ms_);
   }
 
+  bool preserve_subresource_hints() const {
```

Comment:
somewhat confusing with the filter being called `strip_subresource_hints_filter` and the option being called `preserve_subresource_hints`, suggest changing the option to be strip instead of preserve.